rest_framework/fields.py

```diff
@@ -923,7 +927,12 @@ def to_internal_value(self, data):
         Validate that the input is a decimal number and return a Decimal
         instance.
         """
+
+        if self.localize:
+            data = sanitize_separators(data)
```

Comment:
I think this would make slightly more sense after the `smart_text`/`.strip()` line below.